gnucash/report/stylesheets/footer.scm

```diff
@@ -102,15 +102,9 @@
       (N_ "Images")
       (N_ "Heading Alignment") "c" (N_ "Banner for top of report.")
       'left
-      (list (vector 'left
-                    (N_ "Left")
-                    (N_ "Align the banner to the left."))
-            (vector 'center
-                    (N_ "Center")
-                    (N_ "Align the banner in the center."))
-            (vector 'right
-                    (N_ "Right")
-                    (N_ "Align the banner to the right.")))))
+      (list (vector 'left (N_ "Align the banner to the left"))
+            (vector 'center (N_ "Align the banner in the center"))
+            (vector 'right (N_ "Align the banner to the right")))))
```

Comment:
I suggest to keep the short names. Together with the option name and tool tip that's fully self-explanatory.